app/views/pages/attend.html.erb

```diff
@@ -1,4 +1,5 @@
 <h1>Attend Public Page</h1>
+<%= render 'shared/topbanner' %>
 <%= render 'what_you_will_learn' %>
 <%= render 'shared/sponsors' %>
 <%= render 'shared/newsletter'%>
```

Comment:
I put it here in the correct position I think? At the top of the page...? If that creates a conflict that would be easily solvable right? Otherwise we have to change it afterwards anyway?